src/main/java/org/fxmisc/flowless/VirtualFlowContent.java

```diff
@@ -169,6 +170,14 @@ void scrollBreadth(double deltaBreadth) {
         setBreadthOffset(breadthOffset.get() + deltaBreadth);
     }
 
+    void scrollLengthToPixel(double pixel) {
+        setLengthOffset(pixel);
+    }
+
+    void scrollBreadthToPixel(double pixel) {
+        setBreadthOffset(pixel);
+    }
```

Comment:
But you sure have written methods like that. Those are the ones where you don't specify neither of `public`, `protected` or `private` modifier, and they are visible only within the same package.